engine/components/sql/src/main/scala/pl/touk/nussknacker/sql/db/schema/DbMetaDataProvider.scala

```diff
@@ -23,10 +25,28 @@ class DbMetaDataProvider(getConnection: () => Connection) {
       } finally statement.close()
     } finally conn.close()
   }
+
+  def getSchemaDefinition(): SchemaDefinition = {
+    val connection = getConnection()
+    val tables = connection.getMetaData.getTables(null, "PUBLIC", "%", Array("TABLE").map(_.toString))
```

Comment:
Is it DB agnostic? I'm afraid it's not... schema pattern etc. also should probably be configurable?

Reply:
Can we try "connection.getSchema" first? We **have** to use it anyway - or prepend schema to table names...

Reply:
I'll use connection.getSchema and for various schema we'll instruct the user to use connection string schema setup - thank to that we won'nt need to prepend schema in queries